### DISCUSSION TITLE
Error Anycubric Kobra2 Slicer

### DISCUSSION THREAD

**Discussion by Bluppy** [OrcaSlicer/OrcaSlicer#2208] [General]
Hello Guys,

i have a Problem with Orcaslicer Kobra 2 Profile

Relative extruder addressing requires resetting the extruder position at each layer to prevent the loss of floating point accuracy. Add "G92 E0" to layer_gcode.

Orcaslicer 1.7.0 last Version is installed.

<img width="384" alt="Screenshot 2023-09-23 at 14 57 34" src="https://github.com/SoftFever/OrcaSlicer/assets/18369144/890405ea-8277-46e4-8c31-5ffde76dfa7b">

**Comment by The-Biscuit** [comment]
I found out how to get rid of the popup.

![image](https://github.com/SoftFever/OrcaSlicer/assets/146376191/c23d2928-25ea-4e37-b489-1db7cf1cb483)
Click on the box right next to the printer you have selected

![image](https://github.com/SoftFever/OrcaSlicer/assets/146376191/6fab997a-f795-4cd7-8f75-53eb6d2a7c7d)
Make sure you have **advanced** selected in order to see the "Machine G-Code Tab"
Scroll down until you see the "Before Layer Change G-Code"
Then type "G92 E0" Below the text that is already there. 

That resolved the issue for me.

Edit - Let me know if this works for you or not=

**Reply by DEgoogledThankGod** [reply]
Thank you!  I had same problem!  This fixed it for me!  I'm brand new to 3D printing.  SO much to learn holy crap!  Thanks again.

**Reply by AnimaDevil28** [reply]
I fixed my problem. thank you

**Comment by Cpat39** [comment]
Thanks your suggestion did resolve the issue for me.

**Comment by ghost** [comment]
If anyone else is having the same issue, you just need to disable "Use relative E distances" in the printer's Basin information tab 
![изображение](https://github.com/SoftFever/OrcaSlicer/assets/155579219/0aada3db-e956-4033-a562-c93b291c5aea)
